src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -188,6 +189,24 @@ public boolean validate() {
             return false;
         }
 
+        //null element in the values of all the possible keys with DataType as ARRAY
+        JsonNode contentAsNode = getContentAsNode();
+        for (Entry<String, DataType> allowedKey : allowedKeys.entrySet()) {
+            DataType dataType = allowedKey.getValue();
+            if (dataType == DataType.ARRAY && contentAsNode.findValue(allowedKey.getKey()) != null) {
```

Comment:
Avoid double calls to `contentAsNode.findValue()`, also I don't think that `findValue()` is the right method. Use `get()`.

Reply:
fixed